cloudwatchmetrics/src/main/java/software/amazon/awssdk/metrics/internal/cloudwatch/MetricUploaderThread.java

```diff
@@ -53,7 +53,7 @@ public void run() {
                 Iterable<PutMetricDataRequest> requests = qIterator.nextUploadUnits();
                 for (PutMetricDataRequest req : requests) {
                     appendUserAgent(req);
-                    log.debug(req);
+                    log.debug(req.toString());
```

Comment:
Use if debug enabled check here? This might save extra effort for converting req to string if debug mode is off.